docs/Implementation.tex

```diff
@@ -57,7 +72,14 @@ \section{Probleme bei der Implementierung}
 als die restlichen Produktionsvariablen abfragen.
 
 \section{Unit tests}
-
+Unit tests waren ausschlaggebend wichtig für den ersten Teil der Implementierung, bei dem die Controller noch nicht entwickelt waren.
+Entsprechend ist die Testabdeckung sehr hoch. Das Model besitzt eine Zeilenabdeckung von 93\%, ebenso wie die Core-Bestandteile.
+Um die Funktionalität von OPC UA Server und Client zu testen, musste von reinen unit tests abgewichen werden, da der Client keine
```

Comment:
Ich bin mit der Rechtschreibung von **unit test** und **Unit test** nicht einverstanden. Klar, es ist ein englisches Wort, aber es sieht in einem deutschen Text falsch aus. Ich schlage **Unittest** vor.